envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/EnvoyIngressRoutesFactoryTest.kt

```diff
@@ -80,6 +75,26 @@ internal class EnvoyIngressRoutesFactoryTest {
         adminRedirectRoute()
     )
 
+    private val getRoute: Route.() -> Unit = {
```

Comment:
I've seen we have a couple of different styles of asserting routes. I'd love to have one consistent way. What about something like simple methods? :) 
```kotlin
    private fun getRoute(route: Route) {
        route.apply {
            allOpenIngressRoute()
            matchingOnMethod("GET")
            matchingRetryPolicy(retryPolicyProps.perHttpMethod["GET"]!!)
        }
    }
    
    private fun headRoute(route: Route) {
        route.apply {
            allOpenIngressRoute()
            matchingOnMethod("HEAD")
            matchingRetryPolicy(retryPolicyProps.perHttpMethod["HEAD"]!!)
        }
    }
    private fun otherRoute(route: Route) {
        route.apply {
            allOpenIngressRoute()
            matchingOnAnyMethod()
            hasNoRetryPolicy()
        }
    }

    private val ingressRoutes = arrayOf(
            ::getRoute, ::headRoute, ::otherRoute
    )
```

Reply:
I want the same thing. Let's add a ktlint / detekt rule for that. Which style do you guys prefer?